lib/inspec/resources/users.rb

```diff
@@ -425,10 +463,26 @@ def credentials(username)
         multiple_values: false
       ).params
 
+      cmd = inspec.command("echo $((($(date +%s) - $(date -d '#{params["Last password change"]}' '+%s'))/86400))")
+      if cmd.exit_status != 0
+        dayslastset = nil
+      else
+        dayslastset = convert_to_i(cmd.stdout.chomp)
+      end
+
+      cmd = inspec.command("lastb -w -a | awk '{print $1}' | grep #{username} | wc -l")
```

Comment:
I think we could just do `lastb -w -a #{username}|wc -l`

But also, we'll need to subtract 2 from that number. As we'll have a empty line and informational line about btmp.

```
$ sudo lastb -w -a miah
miah     ssh:notty    Fri Sep  6 11:12 - 11:12  (00:00)     ::1
miah     ssh:notty    Fri Sep  6 11:12 - 11:12  (00:00)     ::1
miah     ssh:notty    Fri Sep  6 11:12 - 11:12  (00:00)     ::1

btmp begins Fri Sep  6 11:12:07 2019
```

We'll also need to note that this is not a cumulative number of all bad login attempts from that user. Its only since the creation of that btmp file; because this stores all bad login attempts systems on the public internet might refresh this more often. Some distro's might even recreate this file on reboot. I don't have all the details on the lifecycle of that file, but it might be good to consider.

Reply:
Same as your previous comment -> discussion should be done on issue #4432. I noted that specifics down but there is now way to determinate an all time bad Login Count. Because of the log Rotation and Retention Policy. This Count is Always depending on the local System Setup. (Same like on Windows). But every Administrator should be Aware of this. New ideas or any comment is welcome.

Reply:
Just some additional remarks. I this its better to keep go with "awk&grep" on your System i have to calculate "-2" but Maybe on another distro there are 3 empty lines are an additional info line. This could break the calculation. By using awk and grep i can ensure that it only Counts the lines of badloginattempts. The Performance should have no effect in that case.

Reply:
I don't see any need for the `awk` part. I'd go for grep + wc or even `grep -c` (but I'm not sure how universal the count flag is).

Reply:
And discussion about code should go next to the code so we know what we're talking about.